src/content/en/2020/seo.md

```diff
@@ -33,7 +32,7 @@ Despite its complex multidisciplinary nature, in recent years SEO has evolved to
 {{ figure_markup(
   image="seo-google-trends.png",
   caption="Search Engine Optimization Topic search vs. Pay-Per-click and Social media marketing in Google Trends.",
-  description="TODO",
+  description="TODO.",
```

Comment:
@aleyda can you suggest a description for this figure?

Reply:
@rviscomi Yes! Can it be: "SEO search trend vs. other digital marketing strategies in Google" should I add it directly? Just let me know :)

Reply:
Hey @aleyda that is more like the caption/alt text which has already been provided. The description is a more detailed text for those unable to see the figure. Slightly more relevant for the graphs since we place such emphasis on the data in the project, but still let's put a good one in here. I'd suggest something along the lines of:

```suggestion
  description="Screenshot from Google Trends showing the change in time for three SEO related terms: Search Engine Optimization starts at 25% but shows an increasing importance over time to 75% now. Pay-per-click similarly starts at 25% but started falling in 2009 and looks to be less than 10% now. Social-media marketing starts very small (approximately 5%) and shows some slight growth over time ending up slightly above pay-per-click around the 10% mark.",
```

You can either commit this suggestion, or feel free to edit the text in this branch with your own wording.

Reply:
@bazzadp Ohhhh I see the difference now! Thanks so much Barry for the explanation. Just one thing in your proposed description: 

Instead of: 

"Screenshot from Google Trends showing the change in time for three SEO related terms:"

It would rather be: 

"Screenshot from Google Trends showing the change in time for three digital marketing related terms:"

Thanks a lot again :)

Reply:
Done and committed!

Reply:
Thanks @bazzadp 🙌